gr-uhd/docs/uhd.dox

```diff
@@ -84,25 +84,27 @@ set.
 
 The following command keys are understood by both UHD Source and Sink:
 
-Command name | Value Type   | Description
--------------|--------------|-------------------------------------------------------------
-`chan`       | int          | Specifies a channel. If this is not given, either all channels are chosen, or channel 0, depending on the action. A value of -1 forces 'all channels', where possible.
-`gain`       | double       | Sets the Tx or Rx gain (in dB). Defaults to all channels.
-`freq`       | double       | Sets the Tx or Rx frequency. Defaults to all channels. If specified without `lo_offset`, it will set the LO offset to zero.
-`lo_offset`  | double       | Sets an LO offset. Defaults to all channels. Note this does not affect the effective center frequency.
-`tune`       | tune_request | Like freq, but sets a full tune request (i.e. center frequency and DSP offset). Defaults to all channels.
-`lo_freq`    | double       | For fully manual tuning: Set the LO frequency (RF frequency). Conflicts with `freq`, `lo_offset`, and `tune`.
-`dsp_freq`   | double       | For fully manual tuning: Set the DSP frequency (CORDIC frequency). Conflicts with `freq`, `lo_offset`, and `tune`.
-`direction`  | string       | Used for timed transceiver tuning to ensure tuning order is maintained. Values other than 'TX' or 'RX' will be ignored.
-`rate`       | double       | See usrp_block::set_samp_rate(). *Always* affects all channels.
-`bandwidth`  | double       | See usrp_block::set_bandwidth(). Defaults to all channels.
-`time`       | timestamp    | Sets a command time. See usrp_block::set_command_time(). A value of PMT_NIL will clear the command time.
-`mboard`     | int          | Specify mboard index, where applicable.
-`antenna`    | string       | See usrp_block::set_antenna(). Defaults to all channels.
+Command name | Value Type     | Description
+-------------|----------------|-------------------------------------------------------------
+`chan`       | int            | Specifies a channel. If this is not given, either all channels are chosen, or channel 0, depending on the action. A value of -1 forces 'all channels', where possible.
+`gain`       | double         | Sets the Tx or Rx gain (in dB). Defaults to all channels.
+`freq`       | double         | Sets the Tx or Rx frequency. Defaults to all channels. If specified without `lo_offset`, it will set the LO offset to zero.
+`lo_offset`  | double         | Sets an LO offset. Defaults to all channels. Note this does not affect the effective center frequency.
+`tune`       | tune_request   | Like freq, but sets a full tune request (i.e. center frequency and DSP offset). Defaults to all channels.
+`mtune`      | tune_request_t | Like tune, but supports a full manual tune request as uhd::tune_request_t. Defaults to all channels.
```

Comment:
This will need a little more documentation, since the mapping from tune_request to your dictionary isn't quite obvious.

Reply:
> @segdy separate PRs is super helpful for us. In this case, I'm not sure if we want to backport though, since this goes far beyond a bugfix. It doesn't break API, though, so I could be convinced.

Sounds great. I'll create a new one for gr3.7 ... would be super happy about a backport because that's the main branch I'm still using (I think the changes are small and compact and won't break anything).

> This will need a little more documentation, since the mapping from tune_request to your dictionary isn't quite obvious.

I'll write something more verbose.

Reply:
> > This will need a little more documentation, since the mapping from tune_request to your dictionary isn't quite obvious.
> 
> I'll write something more verbose.

Updated documentation and provided example. Better?